services/spar/src/Spar/Intra/Brig.hs

```diff
@@ -194,14 +194,14 @@ assertIsTeamOwner buid tid = do
   when (statusCode resp >= 400) $ throwSpar SparNotTeamOwner
 
 
--- | Get session token from brig and redirect user past login process.
+-- | Get persistent cookie from brig and redirect user past login process.
 ssoLogin :: (HasCallStack, MonadError SparError m, SAML.HasConfig m, MonadSparToBrig m)
          => UserId -> m SetCookie
 ssoLogin buid = do
   resp :: Response (Maybe LBS) <- call
     $ method POST
     . path "/i/sso-login"
     . json (SsoLogin buid Nothing)
-    . queryItem "persistent" "true"
+    . queryItem "persist" "true"
```

Comment:
Could you add an integration test exercising this function and checking that the resulting cookie is indeed a persistent one, and not a Session cookie (see [CookieType](https://github.com/wireapp/wire-server/blob/df1eb6ab1533c16b1b3b824d3bb3b3027382f115/libs/brig-types/src/Brig/Types/User/Auth.hs#L191-L202))? Such a test would be useful to catch potential future regressions here.

Reply:
will do.  interestingly, it's proving to be tricky to get those tests to pass, so i guess thanks for the suggestion.

Reply:
so, tests are in place now, and they show it's not working yet.  [brig says:](https://github.com/wireapp/wire-server/blob/d67ff751c0ebbf18e7ea826f03da32910454415d/services/brig/src/Brig/User/API/Auth.hs#L189-L195)

```
(Request {requestMethod = "POST", httpVersion = HTTP/1.1, rawPathInfo = "/i/sso-login", rawQueryString = "?persist=true", requestHeaders = [("Host","127.0.0.1:8082"),("Accept-Encoding","gzip"),("Content-Length","60"),("Content-Type","application/json")], isSecure = False, remoteHost = 127.0.0.1:38666, pathInfo = ["i","sso-login"], queryString = [("persist",Just "true")], requestBody = <IO ByteString>, vault = <Vault>, requestBodyLength = KnownLength 60, requestHeaderHost = Just "127.0.0.1:8082", requestHeaderRange = Nothing},True)
[("Set-Cookie","zuid=b2skW9ekmxec0iwsjODIeUhPT6jGRlPJxDNRc8Q3bSRx0DM6nNw5PdXlnxEna_LfswSKv-b_K7nWNWeML-_zAA==.v=1.k=1.d=1542276007.t=u.l=.u=5297de7e-4b92-4028-9959-db374fc20985.r=427442d4; Path=/access; Expires=Thu, 15-Nov-2018 10:00:07 GMT; Domain=127.0.0.1; HttpOnly"),("Content-Type","application/json")]
```

persist flag is passed and accepted, but the resulting token still has a life expectancy of, what? 
 five minutes?